css/program.css

```diff
@@ -354,9 +356,9 @@ img {
     padding: 4.5rem 6rem !important;
   }
 
-  .programlist-subscribe .form-container .subscribe-inputbox{
+  /* .programlist-subscribe .form-container .subscribe-inputbox{
```

Comment:
if this code isn't needed then we can clean this from here instead of commenting,

Reply:
Sure, I'll open a new PR for the new issue.

Reply:
no not this one , it was about this https://github.com/meshery/meshery.io/issues/898 .

You can remove the commented code in this PR itself despite commenting out was my say .